public/styles.css

```diff
@@ -626,3 +621,61 @@ section#pageThree {
 .projectLabel {
     cursor: pointer;
 }
+@media (min-width: 641px) {
+    #side-navbar {
+        z-index: 11;
+        position: fixed;
+        background-color: #fff;
+        top: 0;
+        right: 0;
+        display: block;
+    }
+}
+@media (max-width: 640px) {
+    .loggerBox {
+        border-radius: 2px;
+        background-color: rgba(255, 255, 255, 0.7);
+        max-height: 580px;
+        margin: 80px auto;
+        margin-bottom: 0px;
+        width: 100%;
+    }
+    #dashWrapper {
+        display: flex;
+        flex-direction: column;
```

Comment:
I think `column-reverse` would look better here. This way the chart comes up first. 

We might even consider hiding the query box entirely on smaller screen sizes.

Reply:
Agreed

Reply:
Changed in mp/main-responsive

Reply:
To be addressed in #86